### ISSUE TITLE
Extreme Playlists

### ISSUE THREAD

**Issue by colin969** [FlashpointProject/launcher#127]
All playlists are visible whether Extreme is toggled or not, but games inside are hidden when Extreme is off.

Playlists should be hidden if they are considered Extreme to prevent confusion when having playlists with primarily Extreme games and Extreme games are hidden.

An Extreme field should be added to the Playlist type.

**Comment by badsketch** [comment]
Would like to contribute to this project and take a shot at this 🙂

**Comment by colin969** [comment]
Go ahead :+1: 

Make sure to understand how migrations work when adding a new field to Playlist, if you have any issues or just want to talk then join the Discord server. https://discord.gg/S9uJ794

I really need to add that to contributing.md

**Comment by badsketch** [comment]
Thanks for the heads up. I've joined the server. 

I wanted to ask if this error is still reproducible. I've tried toggling Extreme, but the extreme games still appear inside playlists. I've tried this locally, as well as with the actual client (Windows). Interestingly, this _is_ reproducible on the Mac launcher. Maybe because it's an earlier build? Just wanted to confirm that I'm not going crazy and that I should investigate this as well as part of this issue.
